docs/examples/referential_integrity.yaml

```diff
@@ -47,7 +47,7 @@ rules:
       start_date
     params:
       custom_sql_expr: |-
-        $column in (select unique_key from <dataset_id>.<ref_table_id> where timestamp >= $start_date)
+        $column in (select cast(unique_key as string) from <dataset_id>.<ref_table_id> where timestamp >= "$start_date")
```

Comment:
please escape the table reference in ``, otherwise dataset/table names with BQ control characters can lead to issues.

Reply:
OK.
I had decided not to do that but it's not really worth arguing about it. My thinking was I escape when the reference is through a parameter: `$table_id`, but not when it's a hint to edit the file: `<table_id_here>`, because with the parameter you can put a valid table ID and get an invalid query. If you're expected to edit the file, you need to edit so it's valid and you're editing the query directly.
But it doesn't really matter much, I will escape table names consistently in all cases.